### PR TITLE
Add Sticker Circle, Suit Game, Up Story(owner)

### PR DESCRIPTION
**Pull request by ariffb25**
Result:
image

### CODE REVIEW

**Comment by lilnext01** [comment]
why

**Comment by RC047** [comment]
> 
> why

Jangan pake termux mod tolol

**Comment by ariffb25** [comment]
> 
> why

this is for Baileys, 

As far as I know, open-wa has provided a circle sticker just by filling in the true parameter (I didn't use open-wa, someone told me before)

**Comment by janganlahkau** [comment]
Nurutomo gk update lgi awokawok

**Comment by RC047** [comment]
> Nurutomo gk update lgi awokawok

Lagi Ujian,, dia kelas 9

**Comment by syahrularranger** [comment]
> > 
> > why
> 
> Jangan pake termux mod tolol

Termux emng gitu klo npm i nya di storage

**Comment by RC047** [comment]
> > > 
> > > why
> > 
> > 
> > Jangan pake termux mod tolol
> 
> Termux emng gitu klo npm i nya di storage

Dia termux mod,, soalnya tanda $ nya nama kustom,, udh jelas

File: plugins/stickercircle.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const { sticker } = require('../lib/sticker')
+let fs = require('fs')
+let sharp = require('sharp')
+let handler = async (m, { conn, usedPrefix }) => {
+    const type = Object.keys(m.message)[0]
+    const content = JSON.stringify(m.message)
+    const isMedia = (type === 'imageMessage' || type === 'videoMessage')
+    const isQuotedImage = type === 'extendedTextMessage' && content.includes('imageMessage')
+
+    if (isMedia && !m.message.videoMessage || isQuotedImage) {
```
[the remark this answers is not in the document]
Comment (5M1L3-T3R54K171) [reply plugins/stickercircle.js:11]: Ntaos

**Comment by Kokoronationz** [comment]
UpStory buat apa bang?

**Comment by Nurutomo** [comment]
Termux tidak support module `sharp`

**Comment by Nurutomo** [comment]
barusan kemarin aku buat `upsw.js` tanpa download media...

**Comment by RC047** [comment]
Konflik

**Comment by Nurutomo** [comment]
> Termux tidak support module `sharp`

tidak ada respon :\

**Comment by ariffb25** [comment]
for Termux users, this is not recommended.
for Windows users, please add it yourself if you like it, thanks.

**Pull request closed without merging**